### PR TITLE
Update introduction.stories.mdx

### PR DESCRIPTION
**Pull request by sion-docs**
Suggested some changes to the flow and format of the page to improve readability. Please check to ensure meaning has not been compromised.

### CODE REVIEW

File: packages/theme/stories/introduction.stories.mdx [hunk 1]
```diff
@@ -15,35 +15,21 @@ import { CornerBlock } from "docs/components/CornerBlock";
...
-with non-react components.
-
-#### Theme CSS + Component CSS
-
+### Explaining our approach
+
+For the Toolkit, our design system is built simply from vanilla CSS. There are a number of factors for this decision, including the following:
+* Since we don't rely on any third party styling dependencies, we have been completely free in our approach to styling.
+* There has been a lot of innovation in the CSS world over the last few years and CSS custom properties are one of the most powerful of the new features. Our theming mechanism is built 100% on CSS custom properties. 
+* While CSS-in-JS solutions have improved in terms of performance most still involve some runtime overhead, embracing vanilla CSS removes this overhead
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:23]:
```suggestion
* While CSS-in-JS solutions have improved in terms of performance most still involve some runtime overhead, embracing vanilla CSS removes this overhead.
```

File: packages/theme/stories/introduction.stories.mdx [hunk 2]
```diff
@@ -15,35 +15,21 @@ import { CornerBlock } from "docs/components/CornerBlock";
...
 themselves are styled, it also provides a consistent approach for styling all elements of your application. Most
 applications will probably contain some mix of Toolkit components, plain HTML infrastructure, often for layout purposes,
 custom components you build yourself, and maybe some third party components. You should be able to make use of the
 styling primitives offered by the Toolkit theme to help style all of these. The goal is to make it as easy as possible to
-achieve consistency of styling across the entire application.
-
-The entire solution is built around vanilla CSS (in other words, this is not a `CSS-in-JS` solution). The CSS comprises
-two kinds of declarations:
+achieve consistency of styling across your entire application.
+The CSS comprises the following two kinds of declarations:
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:32]: Is the removal of a new line intentional here?

File: packages/theme/stories/introduction.stories.mdx [hunk 3]
```diff
@@ -124,19 +114,18 @@ And finally, the palette:
 | Positive          | characteristics conveying a positive value or result            | `--uitk-palette-positive-foreground`                     |
 | Rate              | characteristics allowing for something to be rated              | `--uitk-palette-rate-border-undo`                        |
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:115]: In this table appropriate usages don't begin with a capital letter, should they?

File: packages/theme/stories/introduction.stories.mdx (hunk 2 above)
Comment (sion-docs) [reply packages/theme/stories/introduction.stories.mdx:33]: No, fixed now

File: packages/theme/stories/introduction.stories.mdx (hunk 3 above)
Comment (sion-docs) [reply packages/theme/stories/introduction.stories.mdx]: There's no obvious reason why they shouldn't, fixed

**Review by libertymayc — APPROVED** [review]

File: packages/theme/stories/introduction.stories.mdx [hunk 4]
```diff
@@ -15,35 +15,23 @@ import { CornerBlock } from "docs/components/CornerBlock";
...
-applications will probably contain some mix of Toolkit components, plain HTML infrastructure, often for layout purposes,
-custom components you build yourself, and maybe some third party components. You should be able to make use of the
-styling primitives offered by the Toolkit theme to help style all of these. The goal is to make it as easy as possible to
-achieve consistency of styling across the entire application.
-
-The entire solution is built around vanilla CSS (in other words, this is not a `CSS-in-JS` solution). The CSS comprises
-two kinds of declarations:
+The CSS comprises the following two kinds of declarations:
 
 - CSS Custom Properties declarations a.k.a. CSS Variables
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:36]: Nit: missing a full stop here.

File: packages/theme/stories/introduction.stories.mdx [hunk 5]
```diff
@@ -62,14 +50,20 @@ A simple example will help to illustrate some of the key concepts:
...
+- foundations - Foundation tokens are the primitive values for attributes like color, size, shadow etc. Characteristic tokens represent a slightly higher level abstraction - they are groupings of related attributes typically used together, that represent common aspects of a UI component or application.
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:64]: This is more for my knowledge. Here the bullet points don't begin with a capital letter, but there is one after the hyphen. Is this a pattern we will use now?

File: packages/theme/stories/introduction.stories.mdx (hunk 4 above)
Comment (sion-docs) [reply packages/theme/stories/introduction.stories.mdx:36]: It's part of a list and not a full sentence

File: packages/theme/stories/introduction.stories.mdx [hunk 6]
```diff
@@ -208,27 +202,27 @@ The palette token then resolves down to:
...
-1. All UI Toolkit components have a CSS class on the root element, of the form `uitk<ComponentName>`. This is a rule with no exceptions.
-2. The value from a CSS API variable (more on this below) is assigned as a style by using `var(--<variableName>-<...>);`.
-3. A CSS variable reference can define a fallback value to be used if the primary variable does not have a value:
-   var(--primary-variable-value, var(--fallback-value))
-4. The fallback value can equally be a simple constant CSS value
-   var(--primary-variable-value, 12px);
+- All UI Toolkit components have a CSS class on the root element, of the form `uitk<ComponentName>`. This is a rule with no exceptions.
+- The value from a CSS API variable (more on this below) is assigned as a style by using `var(--<variableName>-<...>);`.
+- A CSS variable reference can define a fallback value to be used if the primary variable does not have a value:
+  var(--primary-variable-value, var(--fallback-value))
```
Comment (joshwooding) [inline packages/theme/stories/introduction.stories.mdx:221]: There is an inconsistency where backticks are used for CSS vars in some but not all cases. Plus, I think we are missing some full stops here.

File: packages/theme/stories/introduction.stories.mdx (hunk 4 above)
Comment (sion-docs) [reply packages/theme/stories/introduction.stories.mdx:36]: There are two guidelines which apply here:

- Add a full stop **at the end of complete sentences**—or phrases that complete a sentence that the lead-in phrase begins (example below). - this is the one I followed as it isn't a complete sentence
- Format each bullet point the same way, i.e., write them all as phrases or all as complete sentences. - bearing this one in mind I will need to rewrite the first bullet to make it a complete sentence.

Thanks Josh

File: packages/theme/stories/introduction.stories.mdx (hunk 5 above)
Comment (sion-docs) [reply packages/theme/stories/introduction.stories.mdx:64]: No, this was an oversight on my part after reformatting the paragraph as a bulleted list.

File: packages/theme/stories/introduction.stories.mdx (hunk 6 above)
Comment (sion-docs) [reply packages/theme/stories/introduction.stories.mdx:221]: Thanks, fixed

**Pull request merged**